wgsl/index.bs

```diff
@@ -6668,6 +6668,128 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
+* [=syntax/short_circuit_and_expression|Short-circuit AND=]
+    (can associate with [=syntax/short_circuit_and_expression|self=] and [=syntax/relational_expression|relational=] weakly),
+* [=syntax/binary_or_expression|Binary OR=]
+    (can associate with [=syntax/binary_or_expression|self=] and [=syntax/unary_expression|unary=] weakly),
+* [=syntax/binary_and_expression|Binary AND=]
+    (can associate with [=syntax/binary_and_expression|self=] and [=syntax/unary_expression|unary=] weakly),
+* [=syntax/binary_xor_expression|Binary XOR=]
+    (can associate with [=syntax/binary_xor_expression|self=] and [=syntax/unary_expression|unary=] weakly).
+
+And the following groups do not associate with themselves:
+
+* [=syntax/shift_expression|Shift=]
+    (can associate with [=syntax/unary_expression|unary=] weakly),
+* [=syntax/relational_expression|Relational=]
+    (can associate with [=syntax/additive_expression|additive=] and [=syntax/shift_expression|shift=] weakly).
+
+Associating both group sections above requires parentheses to set the relationship explicitly.
+The following exemplifies where these rules render expressions invalid in comments:
+
+<div class='example wgsl function-scope' heading='Operator precedence corner cases'>
+  <xmp>
+    let a = x & (y ^ (z | w)); // Invalid: x & y ^ z | w
+    let b = (x + y) << (z >= w); // Invalid: x + y << z >= w
+    let c = x < (y > z); // Invalid: x < y > z
+    let d = x && (y || z); // Invalid: x && y || z
+  </xmp>
+</div>
+
+Emergent precedence controls the implicit parentheses of an expression, where the stronger
+binding operator will act as if it is surrounded by parentheses when together with operators
+of weaker precedence. For example, stronger binding multiplicative operators than additive will
+infer `(a + (b * c))` from `a + b * c` expression. Similarly, the emergent associativity controls
+the direction of these implicit parentheses. For example, a left-to-right association will
+infer `((a + b) + c)` from `a + b + c` expression, whereas a right-to-left association
+will infer `(* (* a))` from `* * a` expression.
+
+The following table summarizes operator precedence, associativity, and binding, sorting
+by starting with strongest to weakest. The binding column contains the stronger expression
+of the given operator, meaning, for example, if "All above" is the value, then this operator
+can include any of the stronger expressions. But, for example, if "Unary" is the value,
+then anything weaker than unary but stronger than the operator at row would require parentheses
+to bind with this operator. This column is necessary for linearly listing operators.
+
+<table class='data'>
+  <caption>
+    Operator precedence, associativity, and binding for right-hand side expressions, sorted from strong to weak<br>
+  </caption>
+  <thead>
+    <tr><th>Name
+        <th>Operators
+        <th>Associativity
+        <th>Binding
+  </thead>
+  <tr><td>Parenthesized
```

Comment:
nit: Link names to grammar (as is done on lines 6688+)

Reply:
Done, thanks!